metadata-integration/java/spark-lineage/README.md

```diff
@@ -122,20 +122,21 @@ The Spark agent can be configured using Databricks Cluster [Spark configuration]
+| spark.jars.packages                              | ✅        |         | Set with latest/required version  io.acryl:datahub-spark-lineage:0.8.23                                                                                                                   |
+| spark.extraListeners                             | ✅        |         | datahub.spark.DatahubSparkListener                                                                                                                                                        |
+| spark.datahub.rest.server                        | ✅        |         | Datahub server url  eg:<http://localhost:8080>                                                                                                                                            |
+| spark.datahub.rest.token                         |          |         | Authentication token.                                                                                                                                                                     |
+| spark.datahub.rest.disable_ssl_verification      |          | false   | Disable SSL certificate validation. Caution: Only use this if you know what you are doing!                                                                                                |
+| spark.datahub.metadata.pipeline.platformInstance |          |         | Pipeline level platform instance                                                                                                                                                          |
+| spark.datahub.metadata.dataset.platformInstance  |          |         | dataset level platform instance                                                                                                                                                           |
+| spark.datahub.metadata.dataset.env               |          | PROD    | [Supported values](https://datahubproject.io/docs/graphql/enums#fabrictype). In all other cases, will fallback to PROD                                                                    |
+| spark.datahub.metadata.table.hive_platform_alias |          | hive    | By default, datahub assigns Hive-like tables to the Hive platform. If you are using Glue as your Hive metastore, set this config flag to `glue`                                           |
+| spark.datahub.metadata.include_scheme            |          | true    | Include scheme from the path URI (e.g. hdfs://, s3://) in the dataset URN. We recommend setting this value to false, it is set to true for backwards compatibility with previous versions |
+| spark.datahub.metadata.remove_partition_pattern  |          |         | Remove partition pattern. (e.g. /partition=\d+) It change database/table/partition=123 to database/table                                                                                  |
```

Comment:
Please add the default value! What is the default?

Reply:
Also what are sample values?

Reply:
I don't think it need to have default value. If it is null, It don't change any value.
I thought description is sample value, should I do more? (value: /partition=\d+)